src/components/projects/projectsList.jsx

```diff
@@ -35,26 +47,40 @@ const projectsList = [
     links: [
       {
         href: 'https://github.com/codeforpdx/PASS',
-        icon: <FaGithub size={45} />
+        icon: <FaGithub />
       },
       {
         href: 'https://discord.gg/x6b573et',
-        icon: <FaDiscord size={45} />
+        icon: <FaDiscord />
       },
       {
         href: 'https://passsmartwallet-967e217a2652.herokuapp.com/',
-        icon: <FaEarthAmericas size={45} />
+        icon: <FaEarthAmericas />
       }
     ],
-    techStack: 'SOLID, React, Vite, JSDocs, MUI, NPM, ES Lint'
+    techStack: ['SOLID', 'React', 'Vite', 'JSDocs', 'MUI', 'NPM', 'ES Lint']
   },
   {
     index: 3,
     title: 'CODE PDX',
     description:
       'Code PDX itself is a project that requires constant upkeep and volunteers to maintain. This ranges from project management, assisting with organizing events, or even updating this website! We are constantly improving to better help our community.',
     logo: '/assets/rose_logo.png',
```

Comment:
We still ought to get higher res versions of both PASS and CODE PDX logos. They're grainy on larger viewports.

Reply:
agreed.

Reply:
I have a note on my desk now to get these replaced. I think there was an SVG for both but they got over-written in a push somewhere. If I don't get these to you within a few days please remind me on discord.